multimedia/ffmpeg/060-fix-assembling-with-binutils-2.41.patch

```diff
@@ -0,0 +1,72 @@
+From 25cd95a9dc3510c3cc0d7aad6f9d83f6a1078c7e Mon Sep 17 00:00:00 2001
```

Comment:
Drop this one. Patch under patches folder is just fine

Reply:
Thanks. Just pushed. Let's see...